vimiv/api/settings.py

```diff
@@ -493,9 +493,19 @@ class title:  # pylint: disable=invalid-name
 class metadata:  # pylint: disable=invalid-name
     """Namespace for metadata related settings."""
 
+    # Default sets
+    defaults = [
+        "Make, Model, DateTime, ExposureTime, FNumber, IsoSpeedRatings, FocalLength, LensMake, LensModel, PixelXDimension', 'PixelYDimension, ExposureBiasValue",
+        "ExposureTime, FNumber, IsoSpeedRatings, FocalLength",
+        "Artist, Copyright",
+    ]
+
     # Store the keys as a comma seperated string
-    keys = StrSetting(
-        "metadata.keys",
-        "Make, Model, DateTime, ExposureTime, FNumber, isospeedratings, FocalLength",
-        desc="Define the metadata keys to display",
+    current_keyset = StrSetting(
+        "metadata.current_keyset",
+        defaults[0],
+        desc="Currently displayed metadata keyset",
+        suggestions=defaults,
     )
+
+    keysets: List[str] = defaults
```

Comment:
I would personally prefer a dictionary for this as it:
* Allows the user to keep non-continuous keys, e.g. `1, 2, 3, 8, 9, 0`
* Simplifies keeping defaults and overriding specific options

An option could be:
```
keysets = dict(enumerate(defaults, start=1))
```
where we start with 1 to be consistent with other count-related commands.